htdocs/js/wui.js

```diff
@@ -225,7 +232,6 @@ vz.wui.dialogs.init = function() {
 		try {
 			entity.cookie = Boolean($('#entity-public-cookie').prop('checked'));
 			entity.active = true;
-			entity.subscribe();
 			vz.wui.addEntity(entity);
```

Comment:
weil mit meiner Änderung das `entity.subscribe()` in `addEntity` passiert

Liste kann ich machen.